tests/contrib/aiohttp/test_middleware.py

```diff
@@ -151,3 +151,33 @@ class TestTraceMiddleware(AioHTTPTestCase):
         # applying the middleware twice doesn't add it again
         TraceMiddleware(app, self.tracer)
         eq_(1, len(app.middlewares))
+
+    @unittest_run_loop
+    async def test_exception(self):
+        request = await self.client.request('GET', '/exception')
+        eq_(500, request.status)
+        await request.text()
+
+        traces = self.tracer.writer.pop_traces()
+        eq_(1, len(traces))
+        spans = traces[0]
+        eq_(1, len(spans))
+        span = spans[0]
+        eq_(1, span.error)
+        eq_('error', span.get_tag('error.msg'))
+        ok_('Exception: error' in span.get_tag('error.stack'))
```

Comment:
can you also check if the resource is properly set? something like: https://github.com/DataDog/dd-trace-py/blob/894550e97763d8b2fd702422962d109322ccb2c8/tests/contrib/aiohttp/test_middleware.py#L75